updated to the newest version of the original repository with two bug fix

yangpeiren opened the pull request:
updated to the newest version of the original repository with two bugs fixed.

Bug 1 in opcua/common/structures.py:  

Cause an exception when an uatype is not registered yet.

Bug 2 in opcua/server/address_space.py:

if there are lots of nodes already in address space, a maximum recursion depth exceeded exception will occur, since the lazy node number generation uses always recursion if one number is occupied.
here we get the biggest number from the address space and use it directly.

destogl commented:
I would wait for two bug fixes to be merged and than merge/push upstream directly.

yangpeiren commented:
@destogl they were merged just then, I updated my fork. Now I switched the submodule 'python-opcua' from my fork to the original in the project ros_opcua_impl_python_opcua.
